fury/ui/helpers.py

```diff
@@ -23,6 +22,74 @@ def clip_overflow(textblock, width, side='right'):
     -------
     clipped text : str
         Clipped version of the text.
+    """
+    original_str = textblock.message
+    prev_bg = textblock.have_bg
+
+    is_overflowing, mid_ptr = check_overflow(textblock, width, '...', side)
+
+    if mid_ptr == 0:
+        return original_str
+
+    if not is_overflowing:
+        textblock.have_bg = prev_bg
+        return textblock.message
+
```

Comment:
What happens in all the other cases? for example `is_overflowing= True` and `mid_ptr !=0`?  

Please, can you add a test for this case?

Reply:
Hey @skoudoro, with the lastest commit this check has been removed as the value will always be `False`.